src/GitHook/Composer/Scripts/HookInstaller.php

```diff
@@ -106,4 +113,32 @@ public static function installGitHook(Event $event)
         return true;
     }
 
+    /**
+     * @param \Composer\Script\Event $event
+     *
+     * @return bool
+     */
+    public static function installEcoHooks(Event $event)
+    {
+        $vendorDir = $event->getComposer()->getConfig()->get('vendor-dir');
+
+        $ecoBaseDirectory = $vendorDir . '/spryker-eco/';
+        $hookDirectory = $vendorDir . '/spryker/git-hook/hooks/eco/';
+        $modulesDirs = array_filter(glob($ecoBaseDirectory . '*'), 'is_dir');
+
+        foreach (static::$ecoHooks as $hook) {
+            foreach ($modulesDirs as $dirname) {
+                $src = realpath($hookDirectory . DIRECTORY_SEPARATOR . $hook);
```

Comment:
This line can be moved between the foreach statements as it will not change per module directory.

Reply:
Fixed!